python/paddle/incubate/optimizer/functional/bfgs.py

```diff
@@ -33,63 +33,43 @@ def minimize_bfgs(objective_func,
                   name=None):
     r"""
     Minimizes a differentiable function `func` using the BFGS method.
-    The BFGS is a quasi-Newton method for solving an unconstrained
-    optimization problem over a differentiable function.
-    Closely related is the Newton method for minimization. Consider the iterate 
-    update formula
+    The BFGS is a quasi-Newton method for solving an unconstrained optimization problem over a differentiable function.
+    Closely related is the Newton method for minimization. Consider the iterate update formula:
+
     .. math::
-        x_{k+1} = x_{k} + H \nabla{f},
-    If $H$ is the inverse Hessian of $f$ at $x_{k}$, then it's the Newton method.
-    If $H$ is symmetric and positive definite, used as an approximation of the inverse Hessian, then 
+        x_{k+1} = x_{k} + H_k \nabla{f_k}
+
+    If :math:`H_k` is the inverse Hessian of :math:`f` at :math:`x_k`, then it's the Newton method.
+    If :math:`H_k` is symmetric and positive definite, used as an approximation of the inverse Hessian, then 
     it's a quasi-Newton. In practice, the approximated Hessians are obtained
     by only using the gradients, over either whole or part of the search 
-    history, the former is BFGS.
-
-    Reference:
-        Jorge Nocedal, Stephen J. Wright, Numerical Optimization, Second Edition, 2006.
-        pp140: Algorithm 6.1 (BFGS Method).
-    
-    Following summarizes the the main logic of the program based on BFGS. Note: _k represents value of
-    k_th iteration, ^T represents the transposition of a vector or matrix.
-    repeat
-        p_k = H_k * g_k
-        alpha = strong_wolfe(f, x_k, p_k)
-        x_k+1 = x_k + alpha * p_k
-        s_k = x_k+1 - x_k
-        y_k = g_k+1 - g_k
-        rho_k = 1 / (s_k^T * y_k)
-        V_k^T = I - rho_k * s_k * y_k^T
-        V_k = I - rho_k * y_k * s_k^T
-        H_k+1 = V_k^T * H_k * V_k + rho_k * s_k * s_k^T
-        check_converge
-    end 
+    history, the former is BFGS, the latter is L-BFGS.
+
+    Reference: 
+        Jorge Nocedal, Stephen J. Wright, Numerical Optimization, Second Edition, 2006. pp140: Algorithm 6.1 (BFGS Method).
 
     Args:
-        objective_func: the objective function to minimize. ``func`` accepts
-            a multivariate input and returns a scalar.
-        initial_position (Tensor): the starting point of the iterates. For methods like Newton and quasi-Newton 
-        the initial trial step length should always be 1.0.
+        objective_func: the objective function to minimize. ``objective_func`` accepts a multivariate input and returns a scalar.
+        initial_position (Tensor): the starting point of the iterates. 
         max_iters (int): the maximum number of minimization iterations.
```

Comment:
int -> int, optional
 the maximum number of minimization iterations. ->  the maximum number of minimization iterations.Default value: 50.

Reply:
done